### ISSUE TITLE
Need documentation for syntax of “simplified markup”

### ISSUE THREAD

**Issue by dClauzel** [freenet/plugin-FlogHelper#13]
I am writing a simple flog for playing with FlogHelper, but I am puzzled by the “simplified markup”. I understand it as a watered down markdown, but I cannot find what exactly is supported. For example, I can create paragraphs and lists, but not links or emphasis.

Having a summary of the existing elements with their syntax would be very useful.

**Comment by Thynix** [comment]
Agreed. I went thought everything but lists [here](https://www.reddit.com/r/Freenet/comments/2p08pg/playing_with_floghelper_and_encountering_problems/cmzb8zy) when you asked this on `/r/freenet`. Where would such documentation fit in? Does a "formatting help" link under the text area when using "Simplified markup" make sense?

**Comment by dClauzel** [comment]
Yes, such a link would be good. Another possibility could be to have directly the doc on the right of the textarea: there are plenty of spaces in the edit page.
